src/MemberProfile.js

```diff
@@ -29,92 +30,66 @@ const MemberProfile = ({ user }) => {
             </div>
           </div>
         </div>
-        <div className="user-experience">
-          <div className="contributions-nested">
-            <div className="first-child">
-              <div>
-                <span className="hours-contributed">1</span>
-                <div className="details">hours contributed</div>
+        <HelpUsImplementThis>
+          <div className="user-experience">
+            <div className="contributions-nested">
+              <div className="first-child">
+                <div>
+                  <span className="hours-contributed">1</span>
+                  <div className="details">hours contributed</div>
+                </div>
               </div>
-            </div>
-            <div className="second-child">
-              <div>
-                <span className="projects-completed">5</span>
-                <div className="details">projects completed</div>
+              <div className="second-child">
+                <div>
+                  <span className="projects-completed">5</span>
+                  <div className="details">projects completed</div>
+                </div>
               </div>
-            </div>
-            <div className="last-child">
-              <div>
-                <span className="hours-contributed">35</span>
-                <div className="details">meetups attended</div>
+              <div className="last-child">
+                <div>
+                  <span className="hours-contributed">35</span>
+                  <div className="details">meetups attended</div>
+                </div>
               </div>
             </div>
           </div>
-        </div>
+        </HelpUsImplementThis>
       </div>
 
       <div className="card">
         <div className="skillsleft">
           <h2>Skills</h2>
         </div>
-        <div className="skillsright">
-          <Chip
-            className="skills-button"
-            onRequestDelete={() => {
-              return false;
-            }}
-          >
-            Python
-          </Chip>
-          <Chip
-            className="skills-button"
-            onRequestDelete={() => {
-              return false;
-            }}
-          >
-            SQL
-          </Chip>
-          <Chip
-            className="skills-button"
-            onRequestDelete={() => {
-              return false;
-            }}
-          >
-            Enterprise Geodatabases
-          </Chip>
-        </div>
+        <HelpUsImplementThis>
+          <div className="skillsright">
+            {['Python', 'SQL', 'Enterprise Geodatabases'].map(skill => (
+              <Chip
+                key={skill}
+                className="skills-button"
+                onDelete={() => false}
+                label={skill}
```

Comment:
Could you tell where I can see what key, onDelete(), and label is? Also shouldn't we use onRequestDelete() as above?

Reply:
The api has changed in the new version of the library: https://material-ui-next.com/api/chip/

Reply:
material-ui-next.com is the docs site for the new version of the api